doc/source/whatsnew/v0.25.0.rst

```diff
@@ -82,6 +82,7 @@ Other Enhancements
 - :meth:`DataFrame.query` and :meth:`DataFrame.eval` now supports quoting column names with backticks to refer to names with spaces (:issue:`6508`)
 - :func:`merge_asof` now gives a more clear error message when merge keys are categoricals that are not equal (:issue:`26136`)
 - :meth:`pandas.core.window.Rolling` supports exponential (or Poisson) window type (:issue:`21303`)
+- :meth:`MultiIndex.isin` now fails when args are too short (:issue:`26622`)
```

Comment:
can you move this to bugfixes under MultiIndex

also say ` raises ``ValueError`` ` instead of fails

Reply:
done